src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Legacy/ArrayBuffer.cs

```diff
@@ -32,7 +32,7 @@ internal ArrayBuffer(IntPtr jsHandle) : base(jsHandle)
         /// <summary>
         /// Gets a value indicating whether this ArrayBuffer is view.
         /// </summary>
-        /// <value><c>true</c> if is view; otherwise, <c>false</c>.</value>
+        /// <value><c>true.</c> if is view; otherwise, <c>false.</c>.</value>
```

Comment:
Here and elsewhere: C# booleans should be specified using `see langword`, and the dot should not be part of the wrapped word:

```suggestion
        /// <value><see langword="true"/> if is view; otherwise, <see langword="false"/>.</value>
```